src/components/send-money/SendMoneyAmountInput.js

```diff
@@ -62,9 +67,9 @@ class SendMoneyAmountInput extends Component {
    }
 
    handleChangeAmount = (e, { name, value }) => {
-      const amountStatus = !Number.isInteger(Number(value))
-         ? 'Please enter a whole number.'
-         : value > Number(this.props.amount)
+      const amountStatus = value && !/^[0-9]*(|[.][0-9]{1,5})$/.test(value)
+            ? 'Invalid Input'
+            :  Number(value)  > Number(this.props.amount)
```

Comment:
you probably want to use `BN` for this comparison

Reply:
changed